packages/repository/src/relations/relation.types.ts

```diff
@@ -68,6 +68,50 @@ export interface HasManyDefinition extends RelationDefinitionBase {
   keyTo?: string;
 }
 
+/**
+ * A `hasManyThrough` relation defines a many-to-many connection with another model.
+ * This relation indicates that the declaring model can be matched with zero or more
+ * instances of another model by proceeding through a third model.
+ *
+ * Warning: The hasManyThrough interface is experimental and is subject to change.
+ * If backwards-incompatible changes are made, a new major version may not be
+ * released.
+ */
+export interface HasManyThroughDefinition extends RelationDefinitionBase {
+  type: RelationType.hasMany;
+  targetsMany: true;
+
+  /*
```

Comment:
```suggestion
  /**
```

Two `*` characters are needed to distinguish between plain code comments and tsdoc documentation comments.